GOVERNANCE.md

```diff
@@ -1,3 +1,5 @@
+**Note:** This document is a work in progress
+
```

Comment:
This can be removed. If you want to add "Last updated at DATE" or something similar instead, that'd be ok.

Reply:
Actually it's from the k8s gov doc and I liked the idea that it states we are not set in stone but rather ready to update if needed!
https://github.com/kubernetes/community/blob/master/governance.md

we can remove because date will be visible by the last commit!